spring-context/src/main/java/org/springframework/scheduling/annotation/EnableAsync.java

```diff
@@ -134,7 +134,7 @@
  * setting of the <em>thread name prefix</em> of the {@code Executor}; this is because
  * the {@code <task:executor>} element does not expose such an attribute. This
  * demonstrates how the JavaConfig-based approach allows for maximum configurability
- * through direct access to actual componentry.
+ * through direct access to actual component.
```

Comment:
This is not a typo: "componentry" is semantically correct here.

Although the wording could be improved, the suggested change is grammatically incorrect.